.github/workflows/deploy-workflow.yaml

```diff
@@ -13,11 +13,11 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: '18'
+          node-version: '20'
       - run: npm install
       - run: npm run build
         env:
-          REACT_APP_BITLY_KEY: ${{ secrets.REACT_APP_BITLY_KEY }}
+          VITE_BITLY_KEY: ${{ secrets.VITE_BITLY_KEY }}
       - name: Publish build to github pages
         uses: peaceiris/actions-gh-pages@v3.5.7
         with:
```

Comment:
It won't let me comment on line 25 below, so this is the closest I can get. Anyway, since Vite builds to `dist` and not `build`, I think we need to change `publish_dir` from `build` to `dist`.

Reply:
Oh good catch. I'll update this.

Reply:
Updated in [94697af](https://github.com/FSHSchool/FSHOnline/pull/158/commits/94697af9b14905d2262bd658391a8b7d819c2b16).